contracts/utility/CarbonBatcher.sol

```diff
@@ -166,7 +166,7 @@ contract CarbonBatcher is Upgradeable, Utils, ReentrancyGuard, IERC721Receiver {
      * - the caller is admin of the contract
      */
     function withdrawNFT(uint256 tokenId, address target) external validAddress(target) onlyAdmin nonReentrant {
-        _voucher.safeTransferFrom(address(this), target, tokenId, "");
+        _voucher.transferFrom(address(this), target, tokenId);
```

Comment:
Cause Hedera HTS NFT token doesn't support `safeTransferFrom`, so we must change to use `transferFrom`